openmetadata-docs/content/connectors/database/oracle/airflow.md

```diff
@@ -58,6 +59,12 @@ GRANT CREATE SESSION TO new_role;
 
 ```
 
+2. `GRANT SELECT` on the relevant tables which are to be ingested into OpenMetadata to the user
+```sql
+
+GRANT SELECT ON table_name TO {user | role};
+
+```
```

Comment:
do we have to add something like flush or commit at the end of granting the permissions?

Reply:
no